### ISSUE TITLE
"NaN day history" on try commits

### ISSUE THREAD

**Issue by Zentrik** [rust-lang/rustc-perf#1851]
![image](https://github.com/rust-lang/rustc-perf/assets/19725290/d24654b0-154c-4867-aaec-597e9385d442)
To reproduce see https://perf.rust-lang.org/compare.html?start=1c53407e8c7cc922d718bde61ca34f47b6d2120f&end=c60ecf63ab7a2a0a260b0a8ee02adff4f4dbfe87&stat=instructions:u, and look at any compile time benchmark.

**Comment by Kobzol** [comment]
Yeah this is *definitely* not indented :laughing: @eth3lbert do you want to take a look? If not, I'll fix it.

**Comment by eth3lbert** [comment]
Ok, I'll fix it.
